### PR TITLE
[nameof] Update to 0.10.2

### PR DESCRIPTION
**Pull request by Neargye**
**Describe the pull request**

- #### What does your PR fix?
Update nameof to v0.10.2

Changelog:
https://github.com/Neargye/nameof/releases/tag/v0.10.2

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes 

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
Yes 

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/nameof/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nameof",
-  "version-string": "0.10.1",
+  "version-string": "0.10.2",
```
Comment (Cheney-W) [inline ports/nameof/vcpkg.json:3]:
```suggestion
  "version": "0.10.2",
```

File: ports/nameof/vcpkg.json [hunk 2]
```diff
@@ -1,7 +1,18 @@
 {
   "name": "nameof",
-  "version-string": "0.10.1",
+  "version": "0.10.2",
   "port-version": 1,
```
Comment (Cheney-W) [inline ports/nameof/vcpkg.json:4]:
```suggestion
```
If the `version` is updated, then `port-version` can be deleted.

File: ports/nameof/portfile.cmake [hunk 3]
```diff
@@ -3,25 +3,24 @@
...
         -DNAMEOF_OPT_BUILD_EXAMPLES=OFF
         -DNAMEOF_OPT_BUILD_TESTS=OFF
         -DNAMEOF_OPT_INSTALL=ON
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/nameof TARGET_PATH share/nameof)
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/nameof PACKAGE_NAME nameof)
```
Comment (Cheney-W) [inline ports/nameof/portfile.cmake:21]:
```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/${PORT})
```

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**